docs/source/developers/benchmarks.rst

```diff
@@ -103,6 +123,16 @@ expressions.
     --suite-filter=compute-aggregate --benchmark-filter=Kernel \
     /tmp/arrow-bench*/{WORKSPACE,master}/build
 
+Instead of rerunning benchmarks on comparison, a JSON file (generated by
+``archery benchmark run``) may be specified for the contender and/or the
+baseline.
+
+.. code-block:: shell
+  archery benchmark run --output=baseline.json $HOME/arrow/cpp/release-build
+  git checkout some-feature
+  archery benchmark run --output=contender.json $HOME/arrow/cpp/release-build
+  archery benchmark diff contender.json baseline.json
+
```

Comment:
Thanks for adding this info. I will most likely benefit from the capability of saving benchmark results in JSON files and being able to compare them at will.